<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="settingContents">
         <property name="maximumSize">
          <size>
           <width>128</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="currentRow">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="settingPages">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageConfig">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>331</width>
             <height>81</height>
            </rect>
           </property>
           <property name="title">
            <string>Langauge</string>
           </property>
           <widget class="QComboBox" name="comboBox">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>30</y>
              <width>141</width>
              <height>22</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>101</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Select Langauge:</string>
            </property>
           </widget>
          </widget>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>331</width>
             <height>151</height>
            </rect>
           </property>
           <property name="title">
            <string>E-mail Server</string>
           </property>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>QQ E-mail:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_16">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>60</y>
              <width>54</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>163 E-mail:</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>54</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string>Sina E-mail:</string>
            </property>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="pageHelp">
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>70</y>
             <width>54</width>
             <height>12</height>
            </rect>
           </property>
           <property name="text">
            <string>Help...</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
